### PR TITLE
Development

### PR DESCRIPTION
**Pull request by sharat789**
The calculator page is now responsive with 900px set as the breakpoint, the pie chart has tooltips with labels and amount, added gapping between calculated info which we receive from the backend, replace the background image with a background color which has no opacity and added a vertical line break between input form and calculated info(which does not appear on mobile).  For the application form Jurgis should have reviewed the smaller PR. Good Luck!

### CODE REVIEW

File: src/app/loan-form/loan-form.component.ts [hunk 1]
```diff
@@ -39,10 +50,9 @@ export class LoanFormComponent  {
       ],
     });
 
-
     this.loanForm.get('totalAmount')!.valueChanges.subscribe((totalAmount) => {
       const downPaymentControl = this.loanForm.get('downPayment');
```
Comment (bridzius) [inline src/app/loan-form/loan-form.component.ts:54]: Would `this.downPayment` work here?

File: src/app/loan-form/loan-form.component.ts [hunk 2]
```diff
@@ -78,7 +86,7 @@ export class LoanFormComponent  {
   }
 
   getDownPaymentFloatLabelValue(): FloatLabelType {
-    return this.loanForm.get('totalAmount')!.value || 'auto';
+    return this.loanForm.get('downPayment')!.value || 'auto';
```
Comment (bridzius) [inline src/app/loan-form/loan-form.component.ts:89]:
```suggestion
    return this.downPayment.value || 'auto';
```

File: src/app/loan-form/loan-form.component.ts [hunk 3]
```diff
@@ -78,7 +86,7 @@ export class LoanFormComponent  {
   }
```
Comment (bridzius) [inline src/app/loan-form/loan-form.component.ts:86]: `this.totalAmount.value || 'auto';`

File: src/environments/environment.prod.ts
Comment (bridzius) [inline src/environments/environment.prod.ts:1]: Is this file used for anything anymore?

**Review by bridzius — APPROVED** [review]

File: src/environments/environment.prod.ts
Comment (sharat789) [reply src/environments/environment.prod.ts:1]: I think netlify uses this for `npm run build`. But I can try to keep just environment.ts which has both links for production and development.

Comment (bridzius) [reply src/environments/environment.prod.ts:1]: `npm run build` will use `--configuration=production` by default, which will use `environment.ts`, no?

**Review by NeHerkus — APPROVED** [review]

**Comment by bridzius** [comment]
Can this be merged? Cause there's another PR that will update development soon-ish and will make this even more confusing.

**Pull request merged**